test/cypress/integration/Dashboard/layout-add.spec.js

```diff
@@ -0,0 +1,41 @@
+function isWithinParent(element){
+    assert.isAtMost(element.width(), element.parent().innerWidth());
+    assert.isAtMost(element.height(), element.parent().innerHeight());
+}
+
+function checkCells() {
+    cy.get('.hd-cell').each(cell => {
+        isWithinParent(cell)
+    });
+}
+
+function checkRowsAndCells() {
```

Comment:
Is it needed? Where is an assert for that? Can it be moved to cypress utils file?